src/swsssdk/scripts/sonic-db-cli

```diff
@@ -5,53 +5,55 @@ import swsssdk
 import redis
 import argparse
 from multiprocessing import Pool
+from functools import partial
 
-def ping_unix_path_single_instance(inst_info):
+def handle_single_instance(op, use_unix_socket, inst_info):
     inst_hostname = inst_info['hostname']
-    inst_unix_socket_path = inst_info['unix_socket_path']
-    r = redis.Redis(host=inst_hostname, unix_socket_path=inst_unix_socket_path)
-    rsp = False
-    msg = 'Could not connect to Redis at {}:{}: Connection refused'.format(inst_hostname, inst_unix_socket_path)
-    try:
-        rsp = r.ping()
-    except redis.exceptions.ConnectionError as e:
-        pass
-    return 'PONG' if rsp else msg
-
-def ping_tcp_single_instance(inst_info):
-    inst_hostname = inst_info['hostname']
-    inst_port = inst_info['port']
-    r = redis.Redis(host=inst_hostname, port=inst_port)
+    if use_unix_socket:
+        inst_unix_socket_path = inst_info['unix_socket_path']
+        r = redis.Redis(host=inst_hostname, unix_socket_path=inst_unix_socket_path)
+    else:
+        inst_port = inst_info['port']
+        r = redis.Redis(host=inst_hostname, port=inst_port)
     rsp = False
-    msg = 'Could not connect to Redis at {}:{}: Connection refused'.format(inst_hostname, inst_port)
+    msg = 'Could not connect to Redis at {}:{}: Connection refused'.format(inst_hostname,
+           inst_unix_socket_path if use_unix_socket else inst_port)
     try:
-        rsp = r.ping()
+        if op == 'PING':
+            rsp = r.ping()
+        elif op == 'SAVE':
+            rsp = r.save()
+        elif op == 'FLUSHALL':
+            rsp = r.flushall()
+        else:
+            msg = "operation {} is not supported".format(op)
     except redis.exceptions.ConnectionError as e:
         pass
-    return 'PONG' if rsp else msg
+    return 'OK' if rsp else msg
```

Comment:
>'OK' [](start = 11, length = 4)

This is magic string. Can we use empty string?

Reply:
using None instead which make the code more clean.